File: travel-planner-frontend/src/models/ShareToken.js
export class ShareToken {
    constructor(id, token, tripId, accessType, createdAt, expiresAt, isActive, qrCodeBase64) {
        this.id = id;
        this.token = token;
        this.tripId = tripId;
        this.accessType = accessType;
        this.createdAt = createdAt;
        this.expiresAt = expiresAt;
        this.isActive = isActive;
        this.qrCodeBase64 = qrCodeBase64;
    }
}

File: TravelPlannerApp/SharingService/Models/ShareToken.cs
namespace SharingService.Models
{
    public class ShareToken
    {
        public int Id { get; set; }
        public string Token { get; set; }
        public int TripId { get; set; }
        public string AccessType { get; set; } // "VIEW" ili "EDIT"
        public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
        public DateTime ExpiresAt { get; set; }
        public bool IsActive { get; set; } = true;
    }
}